packages/esm-ward-app/src/ward.resource.ts

```diff
@@ -0,0 +1,35 @@
+import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import type { Encounter } from './types';
+
+export function createEncounter(
```

Comment:
I would have hoped there was a hook for this in ESM Core, but, if not, fine to add here.

Reply:
No, we don't have that in the framework.